app/jobs/gallery_validation_job.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+##
+# Validate that a gallery has only properly displaying images
+# Should an issue be discovered, a slack channel will be notified by sending an email.
+class GalleryValidationJob < ApplicationJob
```

Comment:
I don't see any code actually queueing this job...

Reply:
added a call to clock rb and a job so we can run it manually to jobs.rake
https://github.com/europeana/europeana-portal-collections/pull/1008/commits/88821aa2a0b3a599f60bef62aa6bd1c108259a0a#diff-da3f4822b15eb7097ba27f136e6c8f3c